src/webapp/common.py

```diff
@@ -394,6 +395,14 @@ def wrapped():
     return wrapped
 
 
+def get_timestamp():
+    """Return a monotonic timestamp if available, otherwise a wall-clock timestamp."""
+    if hasattr(time, "monotonic"):
```

Comment:
Making it backwards compatible is a noble goal, but with time.monotonic() being added since Python 3.3 (which has been EoL since 2017), seems fine to just go ahead and use it without a shim, imho.

Reply:
Sure